site/content/docs/latest/reference/developer/issue-triage-process.md

```diff
@@ -60,34 +89,50 @@ Besides this manual triage process, there are two automatic process lead by the
 | 'L'   | A story that requires investigation and possibly will take a person a full 2-week iteration |
 | 'XL'  | A story too big or with too many unknowns. Needs investigation and split into several ones  |
 
-### Priority
+| priority/ |       |       |       |
+| --------- | ----- | ----- | ----- |
+| ⛔️ P0    | 🔴 P1 | 🟠 P2 | 🔵 P3 |
```

Comment:
Might be worth a small description for each:

- P0 - drop everything and fix it
- P1 - ...

Fine either way.